front/app/containers/CustomPageShow/CustomPageHeader/FullWidthBannerLayout.tsx

```diff
@@ -32,10 +28,12 @@ const FullWidthBannerLayout = ({
             data-cy="e2e-header-image-background"
             src={imageUrl || null}
           />
-          <HeaderImageOverlay
-            overlayColor={imageColor}
-            overlayOpacity={imageOpacity}
-          />
+          {overlayColor && typeof overlayOpacity === 'number' && (
```

Comment:
Maybe the same as before would make sense
```suggestion
          {overlayColor && overlayOpacity !== null && (
```